README.rst

```diff
@@ -199,6 +215,19 @@ identity will be tried in sequence until one is able to decrypt the data.
 
 Encrypting with SSH keys via age is not yet supported by sops.
 
+Test with the dev AGE key
+**********************
+
+If you want to test **sops** without having to do a bunch of setup, you can use
+the example files and AGE key provided with the repository::
+
+	$ git clone https://github.com/mozilla/sops.git
+	$ cd sops
+	$ export SOPS_AGE_KEY_FILE="$(pwd)/age/keys.txt"
+	$ sops example.yaml
```

Comment:
I'm a fan of moving the example files to using `age`, but I'd like to move that to separate PR or add a test in CI for this. It feels more likely that we could back out the change to the example files without a test for them, and I don't want to back out your fix for the multiple recipients.

Reply:
I will separate this.
Thanks for your feedback!